.github/workflows/build_permissions_bip.yaml

```diff
@@ -4,37 +4,34 @@ on:
   workflow_dispatch:
     inputs:
       PR_NUMBER:
-        description: 'The number of the PR with the a successful Generate Authorizor run'
-        require: true
+        description: "The number of the PR with the a successful Generate Authorizor run"
+        required: true
       BIP_NUMBER:
-        description: 'The number of the BIP to label everything as'
-        require: true
+        description: "The number of the BIP to label everything as"
+        required: true
```

Comment:
What is this about, why is the key changing names?
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_callinputs

Reply:
I think the change is correct, but it surprises me that a linter is doing it, this is beyond just yaml linting.

Reply:
i changed it, `require` is not correct. my editor threw warnings.

also see your ref; correct keyword is `required`